### ISSUE TITLE
Cannot build vmui as written in README

### ISSUE THREAD

**Issue by dlazerka** [VictoriaMetrics/VictoriaMetrics#2500]
**To Reproduce**

1.  `make vmui-release` as written in app/vmui/README.md

**Expected behavior**
Works

**Logs**
```
docker run --rm \
          --user 1000:1000 \
         --mount type=bind,src="/home/dl/work/vm/valyala/VictoriaMetrics/app/vmui",dst=/build \
         -w /build/packages/vmui \
         --entrypoint=/bin/bash \
         vmui-builder-image -c "npm install && npm run build"
npm notice 
npm notice New minor version of npm available! 8.6.0 -> 8.7.0
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.7.0>
npm notice Run `npm install -g npm@8.7.0` to update!
npm notice 
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @testing-library/react@13.0.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   peer react@">=16.8.0" from @emotion/react@11.8.2
npm ERR!   node_modules/@emotion/react
npm ERR!     peer @emotion/react@"^11.0.0-rc.0" from @emotion/styled@11.8.1
npm ERR!     node_modules/@emotion/styled
npm ERR!       @emotion/styled@"^11.8.1" from the root project
npm ERR!       3 more (@mui/material, @mui/styled-engine, @mui/system)
npm ERR!     peerOptional @emotion/react@"^11.5.0" from @mui/material@5.5.3
npm ERR!     node_modules/@mui/material
npm ERR!       @mui/material@"^5.5.1" from the root project
npm ERR!       2 more (@mui/icons-material, @mui/lab)
npm ERR!     2 more (@mui/styled-engine, @mui/system)
npm ERR!   peer react@">=16.8.0" from @emotion/styled@11.8.1
npm ERR!   node_modules/@emotion/styled
npm ERR!     @emotion/styled@"^11.8.1" from the root project
npm ERR!     peerOptional @emotion/styled@"^11.3.0" from @mui/material@5.5.3
npm ERR!     node_modules/@mui/material
npm ERR!       @mui/material@"^5.5.1" from the root project
npm ERR!       2 more (@mui/icons-material, @mui/lab)
npm ERR!     2 more (@mui/styled-engine, @mui/system)
npm ERR!   15 more (@mui/base, @mui/icons-material, @mui/lab, @mui/material, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.0.0" from @testing-library/react@13.0.0
npm ERR! node_modules/@testing-library/react
npm ERR!   @testing-library/react@"^13.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@18.0.0
npm ERR! node_modules/react
npm ERR!   peer react@"^18.0.0" from @testing-library/react@13.0.0
npm ERR!   node_modules/@testing-library/react
npm ERR!     @testing-library/react@"^13.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/node/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2022-04-24T13_51_07_939Z-debug-0.log
make: *** [app/vmui/Makefile:7: vmui-build] Error 1
```

**Version**
latest master [5d87744](https://github.com/VictoriaMetrics/VictoriaMetrics/commit/5d87744ba2dcf27cdf77bbbebe58e8a38dd0d831)

**Comment by valyala** [comment]
This issue has been fixed in the commit a266d50bedc0b0781bada37acc9fc1525b2684ed . The root cause is that the `node:alpine3.15` base image was updated from nodejs v17 to nodejs v18, while vmui cannot be built with nodejs 18.

@Loori-R , could investigate whether is it possible to upgrade nodejs from v17 to v18 for vmui?

**Comment by Loori-R** [comment]
I'll see what can be done
